fastlane/Fastfile

```diff
@@ -38,28 +38,28 @@ platform :ios do
 
   desc "Builds an ipa for the App Store"
   lane :app_store do |options|
+    UI.user_error!("'SENTRY_AUTH_TOKEN' environment variable should be set to use this lane, see Passbolt") unless !ENV["SENTRY_AUTH_TOKEN"].to_s.empty?
```

Comment:
This may get annoying when building locally. The release playbook specifies that we upload dsyms to GitHub manually, so we could follow the same practice for sentry, however it seems better to just automate this and add one more token (on top of App Store Token)